### ISSUE TITLE
flux-sched travis fails because of no hostlist.h 

### ISSUE THREAD

**Issue by dongahn** [flux-framework/flux-sched#67]
Travis CI for flux-sched master fails with

```
resrc.c:35:40: fatal error: src/common/liblsd/hostlist.h: No such file or directory
```
This was mentioned in #400 of flux-core and we need a resolution to get Travis going again.

**Comment by grondo** [comment]
I have a workaround to propose for now [here](https://github.com/grondo/flux-sched/commit/45d24c56a1c864e087d49e6bccb45bf33214a469) but when tests run in travis on my branch they hang just after the RDL test completes (I think jsc tests are next).

See:
https://travis-ci.org/grondo/flux-sched/jobs/80564135

If you have any ideas let me know

**Comment by grondo** [comment]
nevermind, I see @lipari proposed a better fix

**Comment by lipari** [comment]
This problem was fixed with commit f016072366121939ecd808799facfd1b811a0bf9
